cmd/cmdutils.go

```diff
@@ -170,3 +173,49 @@ func printUnmountedStorage(client *occlient.Client, applicationName string) {
 	}
 	fmt.Println("")
 }
+
+// GetAppName returns application name from the provided flag or if flag is not provided, it will return current application name
+func getAppName(client *occlient.Client, projectName string) string {
+	// applicationFlag is `--application` flag
+	if applicationFlag != "" {
+
+		_, err := application.Exists(client, applicationFlag, projectName)
+		if err != nil {
+			checkError(err, "")
+		}
+		return applicationFlag
+	}
+	applicationName, err := application.GetCurrent(projectName)
+	checkError(err, "unable to get current application")
+
+	return applicationName
+}
+
+// GetProjectName checks whether project flag is provided,
+// if provided, it validates and returns the project name or returns current project name
+func getProjectName(client *occlient.Client) string {
```

Comment:
This function is called `getProjectName`, but also sets the project/namespace in `client`.
Calling this something like setNamespace or something like this might be actually better.

Also, it should explain that in the comment together with saying that it reads `projectFlag` variable as it is not obvious  and it can lead to confusion